extensions/interactions/FractionInput/directives/FractionInputValidationService.js

```diff
@@ -194,8 +207,7 @@ oppia.factory('FractionInputValidationService', [
               if (isEnclosedBy(range, ranges[k])) {
                 var rule2 = answerGroups[ranges[k].answerGroupIndex - 1]
```

Comment:
ditto, maybe call this earlierRule or laterRule as appropriate?

Reply:
done